package.json

```diff
@@ -30,11 +30,19 @@
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.12.13",
+    "@stencil/postcss": "^2.0.0",
```

Comment:
Right now I'm using `postcss` to inline the SVG background images. Coincidentally, it also inlines the `@import` rules.